cg/meta/workflow/taxprofiler.py

```diff
@@ -118,3 +126,58 @@ def config_case(
             header=TaxprofilerSampleSheetEntry.headers(),
         )
         self.write_params_file(case_id=case_id, pipeline_parameters=pipeline_parameters.dict())
+
+    def get_multiqc_per_sample(
```

Comment:
this function is duplicated with the one that get the multiqc values per sample. 
If you split the original function into several functions as Henrik suggested and then it would be easy to generalise it so it can either use case or sample